includes/admin/class-amp-template-customizer.php

```diff
@@ -332,12 +350,181 @@ public function add_customizer_scripts() {
 			'amp-customizer',
 			amp_get_asset_url( 'css/amp-customizer.css' ),
 			[],
-			AMP__VERSION
+			$version
```

Comment:
@pierlon This change isn't right as the `$version` here is for the script not the style.

```suggestion
			AMP__VERSION
```

Reply:
Yea that was not intentional. The browser was not loading the changes I made to that CSS file so I used `$version` to bust the cache.